Event API Routes

tausifmuzaffar opened the pull request:
W-4006595

iamigo commented on api/v1/helpers/verbs/findUtils.js:
```diff
@@ -152,6 +152,10 @@ function toSequelizeWhere(filter, props) {
           const v = filter[key][j];
           if (typeof v === 'boolean') {
             values.push(v);
+          } else if (typeof v === 'number') {
```
Good!

iamigo commented on api/v1/swagger.yaml:
```diff
@@ -6941,6 +6941,148 @@ paths:
+            description: Create event properties
+            properties:
+              log:
+                type: string
```
adding descriptions to each of these params will help api users

iamigo commented on api/v1/swagger.yaml:
```diff
@@ -6941,6 +6941,148 @@ paths:
+    x-swagger-router-controller: events
+    get:
+      summary: Retrieve all events
+      tags: [ events ]
```
pls. add "description" attribute (a more verbose version of the "summary" attribute) to all of these new endpoints

tausifmuzaffar commented:
@annyhe do you mind looking at this review while Ian is on PTO?

tausifmuzaffar commented:
@pallavi2209 could you look at this PR?

tausifmuzaffar commented:
@iamigo @pallavi2209 are there more changes needed here?

iamigo commented:
I had comments about adding description attributes. If you promise to add them in a subsequent PR, I'm prepared to approve this PR.

tausifmuzaffar commented:
@iamigo I had added descriptions to the routes and the post parameters, did you get a chance to see them? Just now i also added description to the response.
